src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import './UserInfo.scss';
+import { User } from '../../types/user';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <a className="UserInfo" href={email}>
```

Comment:
If you want to refer an email what prefix should you use in `href`, attribute?
After you will fix that, all test will be passed 🙂
```suggestion
    <a className="UserInfo" href={email}>
```